BackgroundLocationDemo/location.Android/Services/LocationService.cs

```diff
@@ -38,6 +42,26 @@ public override StartCommandResult OnStartCommand (Intent intent, StartCommandFl
 		{
 			Log.Debug (logTag, "LocationService started");
 
+            		// Check if device is running Android 8.0 or higher and call StartForeground() if so
+			if (Build.VERSION.SdkInt >= BuildVersionCodes.O)
+            		{
+				Notification notification = new NotificationCompat.Builder(this, NOTIFICATION_CHANNEL_ID)
+				    .SetContentTitle(Resources.GetString(Resource.String.app_name))
+				    .SetContentText(Resources.GetString(Resource.String.notification_text))
```

Comment:
Missing the resource `notification_text`

Reply:
Apologizes, missing resource added in commit 3744c2b